polybench-c-openmp/README.md

```diff
@@ -0,0 +1,66 @@
+# Polybench OpenMP TAFFO Benchmarks
+Version 3.2 adapted for OpenMP computation
+
+## Usage
+
+TAFFO must have been compiled and installed, and its binaries must be in the current `$PATH`.
+
+First, do
+```bash
+$ export LLVM_DIR=/path/to/LLVM/install
```

Comment:
can we re-write it by making use of `llvm-config`?

Reply:
I don't get what do you mean by `llvm-config` - isn't `llvm-config`  useful only to check which llvm installation is being exported and will be used?

I copied the steps from the other README in here https://github.com/HEAPLab/TAFFO-test/tree/develop/polybench-c-4.2.1-beta

Reply:
I meant to use
```sh
$ export LLVM_DIR=`llvm-config --prefix`
```

Reply:
`llvm-config` can be used to print out several details related to the version of llvm being used, including the install prefix.
Custom installations of LLVM will have custom `llvm-config` executables.

Reply:
Ahh! Makes sense! Thanks for the clarification, going to fix 👌

Reply:
To be honest I don't know if I like using llvm-config because it forces the benchmark to use the LLVM install currently on the PATH (which might be release LLVM and not the debug LLVM I want to use), and additionally it introduces a discrepancy between the old polybench scripts and the OpenMP scripts.

I would ignore this suggestion for now, and postpone it to later internal evaluation between me, @skeru and @michiari

Reply:
I agree with @danielecattaneo on this one.

Reply:
In principle, you can parametrise the `llvm-config` you want to use and use that one everywhere in TAFFO.
However, I do agree on the fact we should uniform this setting all across the code base and be consistent with it.

We can move this discussion internally and have this PR being merged without this change.